live-examples/css-examples/css/border-color.css

```diff
@@ -0,0 +1,6 @@
+#example-element {
+    background-color: #eee;
+    border: 4px solid;
```

Comment:
I think it might be good to make the border wider than this, to make the color more obvious. Maybe 1em, or is that too much?

Reply:
It would be great to have a bit of padding for this - the text is squished at the top of the box. Maybe just `padding: 1em;`? This applies to the `border-width` example too, now I think of it.